enzy_htp/structure/structure_operation/general.py

```diff
@@ -64,6 +64,24 @@ def remove_non_peptide(stru: Structure) -> Structure:
         ch.delete_from_parent()
     return stru
 
+@dispatch
+def remove_hydrogens(residue: Residue) -> Residue:
```

Comment:
Please update the name in anywhere it is called/referenced. You can use the global search function in VSCode.
![image](https://github.com/ChemBioHTP/EnzyHTP/assets/36098720/4bfceefa-51b8-4bf1-8c55-43127d05490e)